src/services/patientService.ts

```diff
@@ -6,6 +6,15 @@ import { apiSlice } from "./apiSlice";
 
 export const patientService = apiSlice.injectEndpoints({
     endpoints: (builder) => ({
+      getPatientsByStatus: builder.mutation<Patient[], any>({
+        query: (data) => ({
+          url: '/patient/getPatient/filter-by-status',  // Ajusta la URL según tu implementación
```

Comment:
same here you don't need to add the comment

Reply:
just please check this one @ThayliAreliVillegas112  and then you can go ahead with the merge

Reply:
@imadiaz ok, thank you